src/app/calculator/utilities/electricity-reduction/electricity-reduction-form/electricity-reduction-form.component.ts

```diff
@@ -147,4 +154,9 @@ export class ElectricityReductionFormComponent implements OnInit {
     }
     this.calculate();
   }
+
+  toggleUserSelectedHP(){
+    this.data.userSelectedHP = !this.data.userSelectedHP;
```

Comment:
let toggledValue: boolean = !this.form.controls.userSelectedHP.value
this.form.controls.userSelectedHP.patchValue( toggledValue)